add explicit ssh notes about users (#2451)
EnterpriseDB/docs: product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx

1. Create an integration script that connects to every (remote)
PgBouncer host and runs the redirect script. Locate the script at `/usr/edb/efm-4.2/bin/efm_pgbouncer_functions`. Make sure the user

> efm can execute the script, which has the following contents (note that the 'efm' user is ssh'ing as 'enterprisedb' to run the script):

Author: The "must" part doesn't make as much sense to me. This sentence is describing what's happening in the script, not what someone should do if writing their own or something. I just want to call attention to that detail in the script that follows.

> efm can execute the script, which has the following contents (note that the 'efm' user is ssh'ing as 'enterprisedb' to run the script):

Reviewer: How about:  The user efm runs ssh as enterpriseDB to run the script.

For the PgBouncer integration, passwordless `ssh` access is required. There are multiple ways 
to configure `ssh`. Follow your organization's recommended process to 
configure the passwordless `ssh`. For a quick start, you can also follow this example for configuring passwordless `ssh`.

> The 'efm' user will need to be able to ssh as the user running pgbouncer, i.e. the 'enterprisedb' user.

Reviewer: ```suggestion
The user efm user must be able to ssh as the user running PgBouncer, enterprisedb.
```

Author: Conceivably they might install pgbouncer as another user besides 'enterprisedb.' If we say elsewhere that PgBouncer has to be installed/run as enterprisedb then this is ok with me. Definitely need to fix the capitalization for PgBouncer though.

Reviewer: How about: The user efm user must be able to ssh as the user running PgBouncer; for example, enterprisedb.

Author: Great!
